### ISSUE TITLE
web-parser-xml.cgi not configurable for sbcl on Ubuntu 

### ISSUE THREAD

**Issue by dpdoughe** [wdebeaum/WebParser#2]
The CGI file web-parser-xml.cgi  appears to be configured either for openmcl or cmucl.  However, on Ubuntu sbcl is the default Lisp which is what I used to build the DRUM parser.  What would be the correct options to run this cgi under SBCL?

I'm guessing it's something like following, but not experienced enough with Lisp CGI to know for sure:
LISP="sbcl --no-sysinit --no-userinit --disable-debugger --end-toplevel-options -quiet -non-interactive --core ../../etc/lisp/web-parser-xml.image"

**Comment by wdebeaum** [comment]
You probably shouldn't be using web-parser-xml.cgi at all (especially for the drum system, which depends heavily on components outside the Parser proper). You're right that it's pretty much hard coded for CMUCL, because that's what's on the only server where it's used. And I'm merging #3 here, because that's also due to you using this version of the code, using `Makefile` instead of `Makefile-component` (see [this line in README.txt](https://github.com/wdebeaum/WebParser/blob/master/README.txt#L64)). The install locations in `Makefile` are specifically for that one server, which I can't use the regular locations under `$TRIPS_BASE` for because I'm installing on a separate machine via a shared network drive (and I can't do the build on the server because I don't have permission to actually log in there). It's true I could set up some symlinks and redo the code to more closely match the regular layout, but it works now on that server, and I'd rather leave well enough alone.

tl;dr: don't just run `make install` in `WebParser/`. Run `make -f Makefile-component install` instead.
